TMessagesProj/src/main/res/values/cg_strings.xml

```diff
@@ -171,7 +171,7 @@
     <string name="CG_ToSaved">Save message</string>
 
     <string name="CG_FABLeft">\"New chat\" on the left</string>
-    <string name="CG_IntroMessage">An **unofficial** Telegram client modification.\nIt has **additional features** and **improved customization**.</string>
+    <string name="CG_IntroMessage">A **custom** Telegram fork.\nBringing you **additional features** and **improved customization**.</string>
```

Comment:
Also referring to changed above
> I think this string shouldn't be changed in that way

Reply:
Sure, I guess I can leave author's message as is.